### PR TITLE
added slider and reworked form

### LINKED ISSUE #169: Rework Form with slider
1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. Rework appropriate to design `form` with adding slider
3. Fetch from `backend` token max multiplier using this endpoint  GET `/api/get-multipliers`
4. Depends on the value, limit max slider value 
Text me in telegram to get access to figma  `djeck_vorobey1`

### PR DESCRIPTION
**Pull request by ikemHood**
Fix #169

### CODE REVIEW

File: frontend/src/components/multiplier.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    outline: none;
+}
+
+.multiplier-slider::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    appearance: none;
+    width: 20px;
+    height: 20px;
+    border-radius: 50%;
+    background: linear-gradient(to right, #4CAF50, white);
```
Comment (whateverfw) [inline frontend/src/components/multiplier.css:29]: use css variable for all occurences of `#4CAF50`

File: frontend/src/components/multiplier.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    position: relative;
+    padding-top: 25px;
+}
+
+.slider-tooltip {
+    position: absolute;
+    top: 0;
+    transform: translateX(-50%);
+    background: #4CAF50;
+    color: #ffffff;
```
Comment (whateverfw) [inline frontend/src/components/multiplier.css:53]: use css variable

File: frontend/src/hooks/useMaxMultiplier.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+export const useMaxMultiplier = () => {
+  const { data, isLoading, error } = useQuery({
+    queryKey: ['max-multiplier'],
+    queryFn: async () => {
+      const response = await axios.get(`${backendUrl}/api/get-multipliers`);
+      return response.data.multipliers;
+    },
+    staleTime: 60 * 60 * 1000, // 1 hour
+    refetchInterval: 60 * 60 * 1000, // 1 hour
```
Comment (whateverfw) [inline frontend/src/hooks/useMaxMultiplier.js:14]: remove comments, create a separate constant, something like `ONE_HOUR_INTERVAL` instead

File: frontend/src/components/MultiplierSelector.jsx [hunk 4]
```diff
@@ -1,39 +1,58 @@
...
-        {multiplier.recommended && (
-          <div className='recommended'>
-            <p>Recommended</p>
+import React, { useMemo, useCallback, useState } from 'react';
+import { useMaxMultiplier } from 'hooks/useMaxMultiplier';
+import './multiplier.css';
+
+
+const MultiplierSelector = ({ setSelectedMultiplier, selectedToken, sliderValue }) => {
+  const { data } = useMaxMultiplier();
```
Comment (whateverfw) [inline frontend/src/components/MultiplierSelector.jsx:7]: could you please also handle error and loading state in the  component?

File: frontend/src/hooks/useMaxMultiplier.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query';
```
Comment (whateverfw) [inline frontend/src/hooks/useMaxMultiplier.js:1]: The package is not listed in the dependencies, which is causing build issues and failing CI. Could you please fix it?

File: frontend/src/hooks/useMaxMultiplier.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+import axios from 'axios';
+
+const backendUrl = process.env.REACT_APP_BACKEND_URL || 'http://0.0.0.0:8000';
+
+export const useMaxMultiplier = () => {
+  const { data, isLoading, error } = useQuery({
```
Comment (whateverfw) [inline frontend/src/hooks/useMaxMultiplier.js:7]: please use latest version of `TanStack Query` (v5), in which `isLoading` doesn't exist, instead it was renamed  to `IsPending`

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Review by whateverfw — APPROVED** [review]

**Review by djeck1432 — APPROVED** [review]
Great and fast job,
Thank you. 
Looks good to me

**Pull request merged**